src/tests/globalStyles.e2e.ts

```diff
@@ -32,19 +31,78 @@ describe("global styles", () => {
             opacity: opacity
           };
         });
-        expect(noticeAnimation.duration).toEqual("0.3s");
+        expect(noticeAnimation.duration).toEqual("0.15s");
         expect(noticeAnimation.name).toEqual(className.slice(className.indexOf("_") + 2));
         expect(noticeAnimation.opacity).not.toBe("0");
       });
     });
 
-    it("should have initial --calcite-animation-timing CSS Custom Property value", async () => {
-      const page = await newE2EPage({ html: snippet });
-      expect(
-        await page.evaluate(() => {
-          return window.getComputedStyle(document.body).getPropertyValue("--calcite-animation-timing");
-        })
-      ).toBe(" 300ms");
+    it("should set animation duration to 0ms", async () => {
+      const page = await newE2EPage({
+        html: snippet
+      });
+      await page.waitForChanges();
+      await page.$eval("calcite-notice", (element: any) => {
```

Comment:
Type the element and avoid using `any` types whenever possible. This applies to similar functions.